examples/patterns/navigation.html

```diff
@@ -3,6 +3,36 @@
 title: Navigation
 category: _patterns
 ---
+
```

Comment:
Can you remove this file as its duplication of the content in the navigation folder.